app/bin/quasar-describe

```diff
@@ -522,6 +461,9 @@ function printQuasarConfOptions ({ quasarConfOptions }) {
 }
 
 function describe (api) {
+  if (apiParts.docs) {
+    openBrowser(api.meta.docsUrl)
+  }
```

Comment:
Would be good to move this after the switch(). In case it fails it should at least print out the API.